docs/plugins/accepted-plugins.rst

```diff
@@ -0,0 +1,27 @@
+Plugins Accepted by Sylius
```

Comment:
The badge is `Approved by Sylius` so I would keep this naming in the documentation as well :)